### PR TITLE
set up Overview Fragment ui and added necessary files

### PR DESCRIPTION
**Pull request by jjhuert99**
just initial setup

### CODE REVIEW

File: app/src/main/java/com/bps/gotwinter2021/ui/overview/OverviewViewModelFactory.kt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.bps.gotwinter2021.ui.overview
+
+import android.app.Application
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.bps.gotwinter2021.favorites.database.FavoriteDatabaseDao
+
+class OverviewViewModelFactory(
+    private val passedName: String,
+    private val application: Application,
```
Comment (Inasity) [inline app/src/main/java/com/bps/gotwinter2021/ui/overview/OverviewViewModelFactory.kt:10]: You don't need to create a viewmodel factory when you can use the function in ViewModelUtils.kt

Comment (jjhuert99) [reply app/src/main/java/com/bps/gotwinter2021/ui/overview/OverviewViewModelFactory.kt:10]: had not seen the ViewModelUtils file thanks

**Review by Inasity — CHANGES_REQUESTED** [review]
Please use the ViewModelUtils to create the viewmodel factory instead

**Review by Rafa311018 — CHANGES_REQUESTED** [review]
i think you have to replace ViewModel factory with ViewModelUtils

**Pull request closed without merging**